lighthouse-cli/index.js

```diff
@@ -100,7 +104,7 @@ Example: --output-path=./lighthouse-results.html`
   .choices('output', Object.keys(Printer.OUTPUT_MODE))
 
   // default values
-  .default('mobile', true)
+  .default('disable-cpu-throttling', true)
```

Comment:
this default should be in `driver.beginEmulation()` as well, since without it there, anyone using LH as a module will be cpu throttled